src/libs/ontologyService.js

```diff
@@ -7,10 +7,22 @@ export async function searchOntology(obolibraryURL) {
   const params = {id: obolibraryURL};
   try{
     let resp = await axios.get(`${baseURL}search`, {params});
-    return resp.data[0];
+    return resp.data;
   } catch(error) {
     Notifications.showError('Error: Ontology Search Request failed');
   }
 }
 
+export function extractDOIDFromUrl(urls) {
+  const doidArr = [];
+  urls.forEach(url => {
+    const startIdx = url.search(/DOID_\d+/);
+    if (startIdx > -1) {
+      doidArr.push(url.slice(startIdx));
+    }
+  });
```

Comment:
I think this is fine for what we need to do. In the future, it is likely that are DOIDs could have a different prefix. We're hoping to go to the MONDO ontology but we don't have any current plans. If we do, something like https://www.npmjs.com/package/url-parse might be a safer way to grab the identifier which is always the last segment of the url.